### PR TITLE
docs(vue-query): migrate vue-query docs

### PR DESCRIPTION
**Pull request by DamianOsipiuk**
@TkDodo
- added a bunch of content tokens to existing docs.
- replaced `React Query` phrases with `Tanstack Query` where appropriate
- run `prettier` on modified pages
- added vue-query docs

### CODE REVIEW

File: docs/react/guides/filters.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ id: filters
 title: Filters
 ---
 
-Some methods within React Query accept a `QueryFilters` or `MutationFilters` object.
+Some methods within Tanstack Query accept a `QueryFilters` or `MutationFilters` object.
```
Comment (TkDodo) [inline docs/react/guides/filters.md:6]: should it be `TanStack` Query (with an upper case S) everywhere?

Comment (DamianOsipiuk) [reply docs/react/guides/filters.md]: Done ✅

**Comment by DamianOsipiuk** [comment]
@TkDodo Do we need some changes here?

**Comment by TkDodo** [comment]
merged :) do we need to merge the PR in the TanStack.com repo as well?

**Comment by DamianOsipiuk** [comment]
@TkDodo Yes

**Comment by RihanArfan** [comment]
Shouldn't all imports of `vue-query` be changed to `@tanstack/vue-query`?

**Comment by TkDodo** [comment]
> Shouldn't all imports of vue-query be changed to @tanstack/vue-query? I'm more than happy to make a PR if that's the case :)

yes, they should. did we not fix this with this PR ?

**Comment by RihanArfan** [comment]
Looks like it was done in #4728 already, perfect :)

**Pull request merged**